README.md

```diff
@@ -135,6 +135,14 @@ directory.
 To install, just copy this one executable into a directory that is
 part of your `PATH` environment variable.
 
+### Alternatively install via Homebrew
+
+If you prefer using [Homebrew](https://brew.sh):
+
+```
+$ brew install swift-protobuf
+```
```

Comment:
I presume this only installs the command-line tool?

We should be more clear about that.

Reply:
This is part of 'Building and Installing the Code Generator Plugin' section. Also described what this command does.